pinot-plugins/pinot-input-format/pinot-protobuf/pom.xml

```diff
@@ -35,9 +35,16 @@
   <url>https://pinot.apache.org/</url>
   <properties>
     <pinot.root>${basedir}/../../..</pinot.root>
-    <proto.version>3.11.4</proto.version>
+    <kafka.lib.version>2.8.1</kafka.lib.version>
```

Comment:
Will move this and other dependency versions to root pom.xml in another PR. Doesn't fall under the current scope

Reply:
to input-format pom might be better.

Reply:
Since we enforce dependency versions in our project, I think it is better to keep everything in root pom.xml so that there's no chance folks can try with a different version in other modules.

Reply:
sounds good. we are still releasing everything together. we can revisit if we decided to separate release